src/types/user.rs

```diff
@@ -28,6 +28,23 @@ pub struct User {
     pub language_code: Option<LanguageCode>,
 }
 
+impl User {
+    pub fn full_name(&self) -> String {
+        match &self.last_name {
+            Some(last_name) => (self.first_name.clone() + last_name.as_ref()),
```

Comment:
Do you sure that `self.first_name.clone() + last_name.as_ref()` will produce `Hello World`, not `HelloWorld`?

Reply:
fixed